src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -334,63 +315,16 @@ Unix support is disabled by default. See https://aka.ms/systemdrawingnonwindows
     <Compile Include="$(CommonPath)Interop\Windows\Ole32\Interop.IStream.COMWrappers.cs"
              Link="Common\Interop\Windows\Ole32\Interop.IStream.COMWrappers.cs" />
     <Compile Include="System\Drawing\DrawingCom.COMWrappers.cs" />
-    <Compile Include="System\Drawing\Icon.Windows.COMWrappers.cs" />
+    <Compile Include="System\Drawing\Icon.COMWrappers.cs" />
     <Compile Include="System\Drawing\Internal\GPStream.COMWrappers.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsWindows)' == 'true' and '$(TargetFrameworkIdentifier)' != '.NETCoreApp'">
     <Compile Include="$(CommonPath)Interop\Windows\Ole32\Interop.IStream.NoCOMWrappers.cs"
              Link="Common\Interop\Windows\Ole32\Interop.IStream.NoCOMWrappers.cs" />
     <Compile Include="System\Drawing\DrawingCom.NoCOMWrappers.cs" />
-    <Compile Include="System\Drawing\Icon.Windows.NoCOMWrappers.cs" />
+    <Compile Include="System\Drawing\Icon.NoCOMWrappers.cs" />
```

Comment:
Should I keep the `NoCOMWrappers` files? My understanding is that we ship real code only on .NET 6 and 7, both of which have `ComWrappers`.

Reply:
I think we can remove NoCOMWrappers files.

Reply:
Done.